apps/system-e2e/src/support/utils.ts

```diff
@@ -1,14 +1,19 @@
-import PDFDocument from 'pdfkit'
+//import PDFDocument from 'pdfkit'
 import fs from 'fs'
 
 export const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
 
 export function createMockPdf() {
```

Comment:
this could also be done async

```suggestion
export const createMockPdf = async () => {
  await fs.promises.writeFile('./mockPdf.pdf', 'test', (e: any) => {
    throw e
  })

  /*
  const doc = new PDFDocument()
  doc.text('test')
  doc.save()
  doc.pipe(fs.createWriteStream('./mockPdf.pdf'))
  doc.end()
  */
}
```

Reply:
This shouldn't be a final version anyway, Deloitte is working on a true fix.